released/packages/coq-dpdgraph/coq-dpdgraph.0.6.5/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "yves.bertot@inria.fr"
+license: "LGPL 2.1"
+
+homepage: "https://github.com/karmaki/coq-dpdgraph"
+
+build: [
+  ["./configure"]
+  ["echo" "%{jobs}%" "jobs for the linter"]
+  [make]
```

Comment:
I have no idea what the `echo` line above is supposed to do, but aren't parallel builds supposed to be activated?
```suggestion
  [make "-j%{jobs}%"]
```